authui/src/back.ts

```diff
@@ -0,0 +1,7 @@
+import { Controller } from "@hotwired/stimulus";
+
+export class BackButtonController extends Controller {
+  connect() {
+    this.element.classList.add("js");
```

Comment:
Instead of adding the `js` class, port the behavior. That is, show the back button in `connect`. Remove the corresponding CSS that reference `.js` class.